### DISCUSSION TITLE
Welcome to Qwen Discussions!

### DISCUSSION THREAD

**Discussion by jklj077** [QwenLM/Qwen3#961] [Announcements]
## 👋 Welcome!
  We're using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ramlalgodara3431** [comment]
Tell me about this purpose

**Reply by 3805661** [reply]
Hi

**Reply by FreeGodCode** [reply]
qwen3_30b_a3b模型使用vllm框架推理，未设置max_tokens参数对推理结果有影响吗？没有设置的话，模型会按照多少的max_tokens来进行推理呢，是模型部署的max_model_len - prompt_token结果作为默认的推理长度吗？
